docs/k8s/role.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
could this file (or this directory in a readme maybe) have some more pointers/instructions? maybe just a pointer to a KB article?

Reply:
I added a readme file. Will update with a link when we have a KB article